### PR TITLE
tests: Add docker logs, pause and port tests

### LINKED ISSUE #55: Add docker integration tests for logs, pause and port
Add integration tests for:
```
docker logs
docker pause / unpause
docker port
```

### PR DESCRIPTION
**Pull request by chavafg**
Also add a test to verify that apt and dnf work
correctly inside a container.

Fixes #55.

Signed-off-by: Salvador Fuentes <salvador.fuentes@intel.com>

### CODE REVIEW

File: integration/docker/package_manager_test.go [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+		It("should not fail", func() {
+			args = append(args, "--rm", "--name", id, DebianImage, "apt-get", "-y", "update")
+			_, _, exitCode := dockerRun(args...)
+			Expect(exitCode).To(BeZero())
+		})
+	})
+
+	Context("check dnf update", func() {
+		It("should not fail", func() {
+			Skip("Issue: https://github.com/clearcontainers/runtime/issues/868")
```
Comment (jodh-intel) [inline integration/docker/package_manager_test.go]: This raises an interesting question: should this skip only apply to `cc-runtime` (when run as `kata-runtime`)? Although `runv` might have the same limitation, I think we will potentially need a way to skip based on runtime. I've raised #59 on what might be an interesting little project.

File: integration/docker/logs_test.go [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+)
+
+var _ = Describe("logs", func() {
+	var id string
+
+	BeforeEach(func() {
+		id = randomDockerName()
+		_, _, exitCode := dockerRun("-td", "--name", id, Image, "sh", "-c", "'echo hello'")
+		Expect(exitCode).To(Equal(0))
+		// Issue https://github.com/clearcontainers/runtime/issues/375
```
[the remark this answers is not in the document]
Comment (chavafg) [reply integration/docker/logs_test.go]: true, yes we can remove the sleep.

File: integration/docker/package_manager_test.go (hunk 1 above)
Comment (chavafg) [reply integration/docker/package_manager_test.go]: seems like at some point yes, we would need to apply the skip based on the runtime. Thanks for raising the issue.

**Comment by chavafg** [comment]
@grahamwhaley good point. I have raised https://github.com/kata-containers/tests/issues/65 to address your comment :)

**Comment by jodh-intel** [comment]
lgtm

**Comment by GabyCT** [comment]
lgtm

**Pull request merged**